src/test/java/com/wikia/webdriver/common/templates/NewTestTemplateCore.java

```diff
@@ -93,15 +102,11 @@ protected void logOutCustomDriver(WebDriver customDriver) {
   }
 
   protected void stopBrowser() {
-    /*
-     * if (NewDriverProvider.getMobileDriver() != null &&
-     * NewDriverProvider.getMobileDriver().getSessionId() != null) {
-     * NewDriverProvider.getMobileDriver().quit(); }
-     */
     if (driver != null) {
       try {
         driver.quit();
       } catch (Error e) {
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Catch Exception instead of Error. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1181)